### PR TITLE
Gabriel/op 177 UI to view the events created from a given address

### PR DESCRIPTION
**Pull request by gabrielorleans**
Screen Shot 2024-09-04 at 12 42 11

home page developed with:

- grid with events list;
- search bar;
- title.

### CODE REVIEW

File: package-lock.json
Comment (ribeirojose) [inline package-lock.json:1]: what's with the package-lock, guys? 😅

**Comment by luizakp** [comment]
this PR needs a rebase against `main`

File: package-lock.json
Comment (luizakp) [reply package-lock.json:1]: please use only one package manager. Or npm or yarn

File: src/app/home/page.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export default function Home() {
+    return (
+      <main className="flex min-h-screen flex-col items-center justify-between p-24">
+        <h1 className= "text-5xl font-bold mb-6">Explore Your Next Event</h1>
+        <form action="">
+            <input className="w-500px h-12 mt-1 pl-2 border-2 rounded-lg bg-gray-50 focus:ring-2 focus:ring-blue-500" placeholder = "Search" type="text"/> 
+        </form>
+        <div className="grid grid-cols-3 gap-12 mt-10">
+        {
+        [1, 2, 3, 4, 5, 6].map((event) => (
```
Comment (luizakp) [inline src/app/home/page.tsx:15]: we should use the schema structure. Please view https://github.com/bleu/op-bacchus/pull/6/files#r1745515431 or https://github.com/bleu/op-bacchus/pull/8/files#r1745517704

File: src/app/home/page.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+export default function Home() {
+    return (
+      <main className="flex min-h-screen flex-col items-center justify-between p-24">
+        <h1 className= "text-5xl font-bold mb-6">Explore Your Next Event</h1>
+        <form action="">
+            <input className="w-500px h-12 mt-1 pl-2 border-2 rounded-lg bg-gray-50 focus:ring-2 focus:ring-blue-500" placeholder = "Search" type="text"/> 
+        </form>
+        <div className="grid grid-cols-3 gap-12 mt-10">
```
Comment (luizakp) [inline src/app/home/page.tsx:13]: think this is breaking on mobile hehehhehe

File: src/components/Header.tsx [hunk 3]
```diff
@@ -1,20 +1,34 @@
...
+
+const NavItem = ({ href, children }: {href:string, children:ReactNode}) => (
+  <Link href={href}>
+    <div className="w-24 h-16 border-b-2 flex justify-center items-center hover:border-b-4 hover:border-black">
+      {children}
+    </div>
+  </Link>
+);
 
 export function Header() {
```
Comment (luizakp) [inline src/components/Header.tsx]: probably breaking on mobile too

**Review by luizakp — CHANGES_REQUESTED** [review]
please rebase and fix the package situation before merging it. Happy to see you first PR ;)

**Pull request merged**